.lando.yml

```diff
@@ -1,7 +1,7 @@
 name: moose
 recipe: wordpress
 config:
-  php: '8.0'
+  php: '8.1'
```

Comment:
@ChrisMKindred thoughts on moving to 8.2 by default? It's supported/recommended by WP Engine, Pantheon, and VIP now. Also, was there a platform setting in composer.json that needs to be adjusted to match this? I seem to recall Justin F mentioning it recently.

Reply:
We can't go 8.2 yet. Here is the supported version chart from WP. https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/

8.1 is compatible with 8.2. The project should be changed to 8.2 if the host defaults to it but if we start with 8.2 and the host defaults to 8.1 (or 8.0) then it is possible for us to not be backward compatible.

Reply:
We can and should add a platform setting for the composer. That can be a different pr though.